tests/agent/test_cli.py

```diff
@@ -0,0 +1,20 @@
+import asyncio
+import os
+import signal
+from unittest.mock import AsyncMock, patch
+
+from connectors.agent.cli import main
+
+
+@patch("connectors.agent.cli.ConnectorsAgentComponent", return_value=AsyncMock())
+def test_main(patch_component):
+    async def kill():
+        await asyncio.sleep(0.2)
+        os.kill(os.getpid(), signal.SIGTERM)
+
+    loop = asyncio.new_event_loop()
+    loop.create_task(kill())
+
+    main()
+
+    loop.close()
```

Comment:
nothing's asserted here. Intended?

Reply:
Yup. `main` blocks so if the test fails the test will be killed by a timeout. I can add some asserts for log messages too, but not sure it's needed

Reply:
Maybe just a comment then, with what you've written in this thread. 👍